google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStandardRunTask.java

```diff
@@ -51,12 +53,19 @@ public AppEngineStandardRunTask(@NotNull RunConfiguration runConfig, @Nullable S
   public void execute(ProcessStartListener startListener) {
     CloudSdkService sdkService = CloudSdkService.getInstance();
 
-    CloudSdk.Builder sdkBuilder = new CloudSdk.Builder()
+    CloudSdk sdk = new CloudSdk.Builder()
         .sdkPath(sdkService.getSdkHomePath())
         .async(true)
-        .startListener(startListener);
+        .startListener(startListener)
+        .build();
 
-    CloudSdkAppEngineDevServer devServer = new CloudSdkAppEngineDevServer(sdkBuilder.build());
+    try {
+      sdk.validateAppEngineJavaComponents();
+    } catch (AppEngineJavaComponentsNotInstalledException ex) {
+      return;
```

Comment:
In that case, the process returned by startListener to AppEngineExecutor will be null.
We're now checking for null in CloudSdkStartupPolicy. If null, we say app-engine-java is missing and provide an instruction on how to install it.